client/layout/logged-out.jsx

```diff
@@ -118,6 +121,7 @@ export default connect( state => {
 	const isJetpackLogin = currentRoute === '/log-in/jetpack';
 
 	return {
+		currentRoute,
```

Comment:
Just pointing out a minor inconsistency here: we're passing `currentRoute` here and have the relevant `currentRoute === '/jetpack/new'` [check](https://github.com/Automattic/wp-calypso/pull/31306/files#diff-27ba10a603b45adc7e189a25058de88bR53) in the wrapped component, whereas we perform the `currentRoute === '/log-in/jetpack'` [check](https://github.com/Automattic/wp-calypso/pull/31306/files#diff-27ba10a603b45adc7e189a25058de88bR121) here in `connect` and pass the result to the component.

We could consider unfying this by either
- Passing both criteria as props, or
- Only passing `currentRoute` and determining both criteria in the wrapped component.

Nothing major though, obviosuly.

Reply:
Thanks!

I think I'm fine leaving it as-is. Ideally, we would have a better way to add/remove classes.